x-pack/platform/plugins/shared/cloud/server/plugin.ts

```diff
@@ -234,6 +234,27 @@ export class CloudPlugin implements Plugin<CloudSetup, CloudStart> {
                     ),
                   })
                 ),
+                resource_data: schema.maybe(
+                  schema.oneOf([
```

Comment:
IMO, it is worth removing this `schema.oneOf` until we actually need it for something... `@kbn/config-schema` is not the most kind to memory and in Kibana we should take any easy savings we can :D

```suggestion
```

Reply:
maybe just do a `shema.maybe`